api/v1beta1/volumereference.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+// VolumeReference represents the source of a volume to mount.
+type VolumeReference struct {
+	// This must match the Name of a ConfigMap.
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
+	// Path within the container at which the volume should be mounted.  Must
+	// not contain ':'.
+	// +optional
```

Comment:
Can you please mention the default path here so users can easily find it out?

Reply:
done